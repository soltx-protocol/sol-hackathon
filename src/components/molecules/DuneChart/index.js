import React from 'react';
import classnames from 'classnames';

import styles from './index.css';

export const DuneDailyFeeByPool = ({
	className,
	feeAccount = 'DLWewB12jzGn4wXJmFCddWDeof1Ma4cZYNRv9CP5hTvX',
	poolTokenMint = '3e1W6Aqcbuk2DfHUwRiRcyzpyYRRjg6yhZZcyEARydUX',
}) => (
	<iframe
		className={classnames(styles.duneChart, className)}
		src={`https://dune.com/embeds/630914/1176681/3d573057-a39b-4bdf-9f63-abf84db2dd9f?feeAccount=${feeAccount}&poolTokenMint=${poolTokenMint}`}
		title="Top 5 Traders"
	/>
);

export const DuneTop5Traders = ({ className }) => (
	<iframe
		className={classnames(styles.duneChart, className)}
		src="https://dune.com/embeds/630617/1176454/f4732340-aafa-43e7-984a-6efe22f8d9f4"
		title="Top 5 Traders"
	/>
);

const DuneChart = ({
	className,
	title,
	poolAccount = 'EGZ7tiLeH62TPV1gL8WwbXGzEPa9zmcpVnnkPKKnrE2U',
}) => (
	<iframe
		className={classnames(styles.duneChart, className)}
		src={`https://dune.com/embeds/626001/1167393/2d14ec26-c13f-483d-8093-aedc1c69ab18?pool_account=${poolAccount}`}
		title={title}
	/>
);

export default DuneChart;
