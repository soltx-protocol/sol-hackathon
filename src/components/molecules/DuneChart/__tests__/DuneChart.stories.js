import React from 'react';
import { storiesOf } from '@storybook/react';

import DuneChart from 'components/molecules/DuneChart';

const stories = storiesOf('molecules/DuneChart', module);

stories.add('__interactive', () => <DuneChart title="tx-pool" />);
