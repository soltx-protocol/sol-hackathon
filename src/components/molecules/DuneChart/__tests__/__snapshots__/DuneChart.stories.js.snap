// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <iframe
    class="duneChart"
    src="https://dune.com/embeds/626001/1167393/2d14ec26-c13f-483d-8093-aedc1c69ab18?pool_account=EGZ7tiLeH62TPV1gL8WwbXGzEPa9zmcpVnnkPKKnrE2U"
    title="tx-pool"
  />
</DocumentFragment>
`;
